src/pages/components/select.mdx

```diff
@@ -22,12 +22,12 @@ components:
   <Misuse>
     <ul>
       <li>
-        To filter a large list of options, consider an{' '}
-        <a href="/components/autocomplete">Autocomplete</a>
+        To filter a large list of options, use{' '}
+        <Markdown>[Autocomplete](/components/autocomplete)</Markdown> instead
       </li>
       <li>
-        To make multiple selections from a list, use a{' '}
-        <a href="/components/multiselect">Multiselect</a>
+        To make multiple selections from a list, use{' '}
+        <Markdown>[Multiselect](/components/multiselect)</Markdown> instead
```

Comment:
@rossmoody the expected JSX formatting would be to do something similar to:

```jsx
<li><Markdown>
  words go here
</Markdown></li>
```

In this way, you set up the entire block to contain whatever markdown is needed either now or in the future.

Reply:
Wonderful, ty.